### PR TITLE
Preliminary pipeline docs

### PR DESCRIPTION
**Pull request by jdesanto**
Draft pipelines documentation

### CODE REVIEW

File: docs/source/user_guide/jobs/data_science_job.rst [hunk 1]
```diff
@@ -99,7 +96,7 @@ All of these runtime options allow you to configure a `Data Science Conda Enviro
     runtime = (
         ScriptRuntime()
         .with_source("oci://bucket_name@namespace/path/to/script.py")
-        .with_service_conda("tensorflow28_p38_cpu_v1")
+        .with_service_conda("tensorflow26_p37_cpu_v2")
```
Comment (darenr) [inline docs/source/user_guide/jobs/data_science_job.rst:99]: should be showing current conda packs

File: docs/source/user_guide/jobs/data_science_job.rst [hunk 2]
```diff
@@ -123,7 +120,7 @@ You can also configure the environment variables, command line arguments, and fr
...
```
Comment (darenr) [inline docs/source/user_guide/jobs/data_science_job.rst:123]: should be using current conda packs

File: docs/source/user_guide/jobs/data_science_job.rst [hunk 3]
```diff
@@ -249,7 +246,7 @@ Here is an example of a YAML file representing the job defined in the preceding
         type: script
         spec:
           conda:
-            slug: tensorflow28_p38_cpu_v1
+            slug: tensorflow26_p37_cpu_v2
```
Comment (darenr) [inline docs/source/user_guide/jobs/data_science_job.rst:249]: should be using current conda packs

File: docs/source/user_guide/jobs/run_notebook.rst [hunk 4]
```diff
@@ -34,7 +36,7 @@ Python
                 path="https://raw.githubusercontent.com/tensorflow/docs/master/site/en/tutorials/customization/basics.ipynb",
                 encoding='utf-8'
             )
-            .with_service_conda("tensorflow28_p38_cpu_v1")
+            .with_service_conda(tensorflow26_p37_cpu_v2")
```
Comment (darenr) [inline docs/source/user_guide/jobs/run_notebook.rst:39]: same comment

File: docs/source/user_guide/jobs/run_notebook.rst [hunk 5]
```diff
@@ -84,7 +86,7 @@ You could use the following YAML to create the job:
 	    spec:
           notebookPathURI: /path/to/notebook
 	      conda:
-            slug: tensorflow28_p38_cpu_v1
+            slug: tensorflow26_p37_cpu_v1
```
Comment (darenr) [inline docs/source/user_guide/jobs/run_notebook.rst:89]: this looks like we've regressed this notebook, maybe instead use head rather than branch

**Review by darenr — COMMENTED** [review]
should also have @lu-ohai take a look at the docs for correctnes

File: docs/source/user_guide/pipeline/pipeline.rst [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+
+
+DAG
+^^^
+
+DAG is used to define the dependencies between the steps. 
+
+- ``>>`` denotes the tasks running in sequence, ``A >> B`` means that A is followed by B.
+- ``()`` denotes the tasks running in parallel.
+- If DAG not provided, all the steps will run in parallel.
```
Comment (lu-ohai) [inline docs/source/user_guide/pipeline/pipeline.rst:22]: `If DAG is not provided`

**Comment by jdesanto** [comment]
Merged via a separate PR.

**Pull request closed without merging**